### DISCUSSION TITLE
all of my clients are offline - x509: certificate has expired or is not yet valid

### DISCUSSION THREAD

**Discussion by Maier-A** [amidaware/tacticalrmm#1130] [Q&A] [answered]
Since 2022-05-12 all of my clients are reporting offline
the agent.log shows x509: certificate has expired or is not yet valid

updates today to latest version - same error

which certificate is meant here?
my let´s encrypt certificates on my rmm server are all valid and ahve been renewed 2022-04-21

thanks for help!

**Comment by dinger1986** [comment]
Yes it's the letsencrypt certificate, did you run `update.sh --force` after renewing it?

**Comment by silversword411** [comment, marked as answer]
...as in follow the instructions: https://docs.tacticalrmm.com/update_server/#keeping-your-lets-encrypt-certificate-up-to-date

**Comment by Maier-A** [comment]
did that, thanks, works, clients online again
